plugins/rollbar_dashboards/app/views/rollbar_dashboards/_deploy.html.erb

```diff
@@ -0,0 +1,23 @@
+<% unless deploy.project.rollbar_dashboards_settings.empty? || !deploy.succeeded?
```

Comment:
prefer to use if instead of unless with multiple conditions

`if settings.any? && deploy.succeeded?`